ecs-init/config/common.go

```diff
@@ -99,3 +111,42 @@ func DockerUnixSocket() (string, bool) {
 	// eg: Docker daemon restart
 	return "/var/run", false
 }
+
+// Find AZ information from Metadata. If error return a blank result
+func EC2MetadataAZ () string {
```

Comment:
1. `EC2MetadataAZ` is global scoped. It doesn't need to be
2. The comment needs to start with the method name as per my previous comment